mcp-registry/servers/f1-mcp-server.json

```
@@ -0,0 +1,283 @@
+{
+    "name": "f1-mcp-server",
+    "display_name": "Formula One MCP Server",
+    "description": "A Model Context Protocol (MCP) server that provides Formula One racing data.",
+    "repository": {
+        "type": "git",
+        "url": "https://github.com/Machine-To-Machine/f1-mcp-server.git"
+    },
+    "homepage": "https://github.com/Machine-To-Machine/f1-mcp-server",
+    "author": {
+        "name": "Machine To Machine"
+    },
+    "license": "MIT",
+    "categories": [
+        "Productivity",
+        "Knowledge Base",
+        "MCP Tools"
```

```suggestion
        "Web Services"
```
Think this should be Web service. MCP Tools means the server is for mcp itself, not for a specific field.